proxy/InkAPI.cc

```diff
@@ -2208,6 +2208,20 @@ TSUrlPathSet(TSMBuffer bufp, TSMLoc obj, const char *value, int length)
   return URLPartSet(bufp, obj, value, length, &URL::path_set);
 }
 
+void
+TSUrlRemoveHostName(TSMBuffer bufp, TSMLoc obj)
```

Comment:
Most of the tests for TS APIs can be found in ```proxy/InkAPITest.cc```, so you can follow the examples there.